### PR TITLE
Feature/functional-pages-text

### PR DESCRIPTION
**Pull request by PotSchmoell**

### CODE REVIEW

**Review by SandroGlueck — APPROVED** [review]

**Comment by PotSchmoell** [comment]
@SandroGlueck I fixed a few issues in this PR. Can you have a quick look at those before we merge this one?
Sorry, I missed that @Th3da had already approved - If I would have realized that, I would have made a separate PR for the recent changes.

File: src/app/core/components/community-guidelines/community-guidelines.component.scss [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
-  flex-direction: column;
+@import '../../../../assets/scss/initializer';
+
+
+.terms-container{
+  h1 {
+    font-size: 52px;
+    font-weight: 700;
+    color: $color__share-section-heading;
+  }
```
Comment (SandroGlueck) [inline src/app/core/components/community-guidelines/community-guidelines.component.scss:9]: Did you test those styles on mobile aswell? 
Seems like there are missing media queries.

File: src/app/my-profile/components/my-profile/my-profile.component.html [hunk 2]
```diff
@@ -7,9 +7,17 @@
...
+        persönlich wichtig sind. Du findest hier deine persönlichen Statistiken, kannst deine gelösten Fälle ansehen,
+        neue Fälle bearbeiten, durchs Archiv stöbern und dein Profil verwalten. Scroll doch einfach mal nach unten und
+        probier alles selbst aus.
+
+
+        <!-- Oben auf der Seite siehst du deine Statistiken und eine Übersicht über gelöste Fälle,
+        an denen du beteiligt warst. Darunter kannst du Fälle bearbeiten. Wähle dazu einfach einen Fall aus, der dich
+        interessiert, klicke auf "Details ansehen" und starte dann mit der Bearbeitung.
+        
+        Außerdem kannst du hier durch das Archiv gelöster Fälle stöbern und dein Profil verwalten. -->
```
Comment (SandroGlueck) [inline src/app/my-profile/components/my-profile/my-profile.component.html:20]: Is that necessary?

File: src/app/services/rating.service.ts [hunk 3]
```diff
@@ -33,7 +33,7 @@ export class RatingService {
       lower: 83,
       upper: 100,
       text: 'Dieser Fall ist vertrauenswürdig',
-      info: 'Du kannst den Informationen in diesem Fall uneingeschränkt vertrauen.',
+      info: 'Du kannst den Informationen in diesem Fall höchstwahrscheinlich vertrauen.',
```
Comment (SandroGlueck) [inline src/app/services/rating.service.ts:36]: Thanks :D

File: src/app/review-item/components/review-success-page/review-success-page.component.html [hunk 4]
```diff
@@ -46,7 +48,8 @@ <h4>Erfahrungspunkte erhalten</h4>
                 </div>
                 <div class="xp__score">
                   <i class="fas fa-stars"></i>
-                  <span>{{ user.experience_points * 100 }} / {{ (user.experience_points + user.exp_needed) * 100 }} XP</span>
+                  <span>{{ user.experience_points * 100 }} / {{ (user.experience_points + user.exp_needed) * 100 }}
+                    XP</span>
```
Comment (SandroGlueck) [inline src/app/review-item/components/review-success-page/review-success-page.component.html:52]: Seems like a weird line break to me

**Review by SandroGlueck — CHANGES_REQUESTED** [review]

File: src/app/my-profile/components/my-profile/my-profile.component.html (hunk 2 above)
Comment (PotSchmoell) [reply src/app/my-profile/components/my-profile/my-profile.component.html:20]: Of course not, I removed it.

File: src/app/review-item/components/review-success-page/review-success-page.component.html (hunk 4 above)
Comment (PotSchmoell) [reply src/app/review-item/components/review-success-page/review-success-page.component.html:52]: You're right. Looks like autoformat didn't work too well on this one. I removed the linebreak

File: src/app/core/components/community-guidelines/community-guidelines.component.scss (hunk 1 above)
Comment (PotSchmoell) [reply src/app/core/components/community-guidelines/community-guidelines.component.scss:9]: You're right. Fixed it and added a little padding on mobile.

**Review by SandroGlueck — APPROVED** [review]

**Pull request merged**